PokeStar/PokeStar/DataModels/Raid.cs

```diff
@@ -1,131 +1,120 @@
 ﻿using System;
+using System.Linq;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using Discord.WebSocket;
 using PokeStar.ConnectionInterface;
 
 namespace PokeStar.DataModels
 {
    class Raid
    {
-      private const int playerLimit = 20;
-      private const int maxInvites = 10;
       public string Location { get; set; }
       public string Time { get; set; }
       public short Tier { get; set; }
       public RaidBoss Boss { get; private set; }
-      public int PlayerCount { get; private set; }
-      public int HereCount { get; private set; }
-      public Dictionary<SocketGuildUser, int> Attending { get; private set; }
-      public Dictionary<SocketGuildUser, int> Here { get; private set; }
-      public Dictionary<SocketGuildUser, int> Invite { get; private set; }
+      public List<RaidGroup> Groups { get; private set; }
+      public List<string> RaidBossSelections { get; set; }
       public DateTime CreatedAt { get; private set; }
+      private Dictionary<SocketGuildUser, int> Invite { get; set; }
```

Comment:
couldnt this just be a list? the party is always going to be of size 1

Reply:
Changed to a list of socketguildusers. GetReadonlyInvite will still return an immutable dictionary with socketguilduser, int key value pairs to allow for calling of BuildPlayerList(ImmutableDictionary<SocketGuildUser, int> players) in RaidCommands.